### PR TITLE
Tkt 00037 - tabbed page navigation

### PR DESCRIPTION
**Pull request by Android63**
screen shot 2018-04-11 at 12 23 15

### CODE REVIEW

**Comment by Android63** [comment]
I've simplified this a lot @cehwitham. It now works as follows:

Child pages are added by admins using the TabbedPage template, assigned to a Parent page and given an order in which to display:
screen shot 2018-04-12 at 14 40 39

Links, in menus or elsewhere, are created by admins to the _first child page_ rather than to the parent page:
screen shot 2018-04-12 at 14 41 33

The result is the first child page is opened as a tab when the user clicks the link:
screen shot 2018-04-12 at 14 40 04

Typically, the first child page will be titled something like "overview".

**Comment by Android63** [comment]
I've changed this so it now displays the main parent page under the first tab, labelled "Overview", with child pages in the remaining tabs. Also added an extra level to breadcrumbs to include parent pages.

Initial (parent) page:
screen shot 2018-04-13 at 11 56 52

Child page:
screen shot 2018-04-13 at 11 57 11

@cehwitham could you review please?

File: inc/breadcrumbs.php [hunk 1]
```diff
@@ -33,6 +33,11 @@ function nightingale_breadcrumb() {
             $category_name = $category->name;
             echo $category_link.'" class="c-breadcrumb__link">'.$category_name.'</a></li>';
           }
+
+					// If this is a child page, add a link to its parent
```
[the remark this answers is not in the document]
Comment (Android63) [reply inc/breadcrumbs.php]: Good point. It now works recursively and will show all the levels of ancestors.

screen shot 2018-04-13 at 13 57 45

File: template-parts/content-tabs.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+		 $post_parent = $post->post_parent;
+	 }
+
+	 // Finish building and displaying Overview tab
+	 $link .= '" href="';
+	 $link .= get_permalink($post_parent);
+	 $link .= '"><span class="c-sprite  c-sprite--home  c-tabs__icon"></span><span class="c-tabs__text">';
+	 $link .= 'Overview';
+	 $link .= '</span></a></li>';
+	 echo $link;
```
Comment (cehwitham) [inline template-parts/content-tabs.php]: Should we check if there are other tabs to be output before outputting an Overview tab?

File: template-parts/content-tabs.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		 'depth' => 1,
+		 'post_status' => 'publish',
+		 'meta_query' => array(
+			 array(
+				 'key' => '_wp_page_template',
+				 'value' => 'page-tabbed.php',
+			 ),
+		 ),
+	 );
+	 $my_query = new WP_Query($args);
```
Comment (cehwitham) [inline template-parts/content-tabs.php]: Can we use a better, more descriptive, prefix than 'my'?

File: template-parts/content-tabs.php (hunk 2 above)
Comment (Android63) [reply template-parts/content-tabs.php]: Agreed and done.

File: template-parts/content-tabs.php (hunk 3 above)
Comment (Android63) [reply template-parts/content-tabs.php]: Agreed and done.

**Pull request merged**